### PR TITLE
[Feature Store] Fixing bug when using OneHotEncoder on category with int values

### PR DESCRIPTION
**Pull request by davesh0812**

### CODE REVIEW

**Review by guy1992l — CHANGES_REQUESTED** [review]
minor change and its good to merge in my eyes.

File: mlrun/feature_store/steps.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import pandas as pd
 from storey import MapClass
 
+from mlrun.errors import MLRunInvalidArgumentError
```
Comment (Tankilevitch) [inline mlrun/feature_store/steps.py:9]: We are trying to use less `from x import y` but rather use `import x`, this reduces the possibility for circular imports as well as makes it better to understand to where certain classes are part of.
```suggestion
import mlrun.errors
```

File: tests/system/feature_store/test_feature_store.py [hunk 2]
```diff
@@ -2606,6 +2606,60 @@ def test_set_event_with_spaces_or_hyphens(self):
 
         assert df_res.equals(expected_df)
 
+    def test_onehot_with_int_values(self):
+        from mlrun.feature_store.steps import OneHotEncoder
```
Comment (Tankilevitch) [inline tests/system/feature_store/test_feature_store.py:2610]: why you import inside the function?

File: tests/system/feature_store/test_feature_store.py [hunk 3]
```diff
@@ -2606,6 +2606,60 @@ def test_set_event_with_spaces_or_hyphens(self):
...
+        )
+        try:
+            data_set.graph.to(OneHotEncoder(mapping=one_hot_encoder_mapping))
+            data_set.set_targets()
+            fs.ingest(data_set, data, infer_options=fs.InferOptions.default())
+        except ValueError:
+            # suppose to catch this error
+            return
+        assert False
+
```
Comment (Tankilevitch) [inline tests/system/feature_store/test_feature_store.py:2662]:
```suggestion
        with pytest.raises(ValueError):
            data_set.graph.to(OneHotEncoder(mapping=one_hot_encoder_mapping))
            data_set.set_targets()
            fs.ingest(data_set, data, infer_options=fs.InferOptions.default())
```

File: tests/system/feature_store/test_feature_store.py [hunk 4]
```diff
@@ -2606,6 +2606,60 @@ def test_set_event_with_spaces_or_hyphens(self):
...
+        expected_df = pd.DataFrame(
+            list(zip([1, 1, 0, 1], [0, 0, 1, 0], lst_3)),
+            columns=["workclass_0", "workclass_1", "age"],
+        )
+
+        assert df_res.equals(expected_df)
+
+    def test_onehot_with_array_values(self):
+        from mlrun.feature_store.steps import OneHotEncoder
```
Comment (Tankilevitch) [inline tests/system/feature_store/test_feature_store.py:2639]: same

**Review by Tankilevitch — CHANGES_REQUESTED** [review]
Looks good, added minor comments

**Review by Tankilevitch — APPROVED** [review]

**Pull request merged**